src/forward.rs

```diff
@@ -128,26 +134,26 @@ impl Forward {
         });
 
         loop {
-            let (stream, addr) = listener.accept().await?;
+            let (client, client_addr) = listener.accept().await?;
             let remote = TcpStream::connect(&self.remote_addrs[0]).await?;
```

Comment:
use `client_stream` and `remote_stream`, instead of `client` and `remote`, which will be confusing